sources/osg/RenderStage.js

```diff
@@ -71,6 +71,17 @@ define( [
         getCamera: function () {
             return this.camera;
         },
+        getPositionnedAttribute: function ( node ) {
```

Comment:
Sorry to bother, but @Kuranes shouldn't it be 'positioned' instead of 'positionned'?